bin/millau/node/src/chain_spec.rs

```diff
@@ -57,10 +58,11 @@ where
 }
 
 /// Helper function to generate an authority key for Aura
-pub fn get_authority_keys_from_seed(s: &str) -> (AccountId, AuraId, GrandpaId) {
+pub fn get_authority_keys_from_seed(s: &str) -> (AccountId, AuraId, BeefyId, GrandpaId) {
```

Comment:
nit: Any reason to add `BeefyId` before `GrandpaId`? Somehow feels more natural to have it at the end, but I guess you preferred to have things sorted?

Reply:
Makes sense!

Reply:
Makes sense, although we have to be careful with such changes for running networks - it's easier to migrate if we add something at the end than to change the ordering.